go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/aliyun/aliyun-oss-go-sdk v2.0.1+incompatible
 	github.com/aws/aws-sdk-go v1.21.10
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
-	github.com/gardener/gardener v0.0.0-20191025063857-c27ebd04a82b
+	github.com/gardener/gardener v0.0.0-20191025121949-82369df93f78
```

Comment:
Wait a sec, why is all of this reverted? Please use the latest `master` branch.

Reply:
I have used @ialidzhikov commit with the fix: https://github.com/gardener/gardener/commit/82369df93f78e7b585efe98911a73b80046038a6

We used for the extension so far this one: https://github.com/gardener/gardener/commit/c27ebd04a82b4ffc3830aef750163c058ba2d4fc

`82369df93f78` is just a few hours newer. I could also switch to the most recent master branch of Gardener.